examples/cloudflare-workers/web/README.md

```diff
@@ -0,0 +1,53 @@
+# Using Momento inside a Cloudflare Worker
```

Comment:
nit - I'd like for our users to be able to make a reasonable guess as to what the difference between the two examples is when they navigate to the cloudflare-directory. Maybe we should call the directories `http-api` and `web-sdk`.